lib/pbench/cli/agent/commands/tools/clear.py

```diff
@@ -25,54 +26,86 @@ def __init__(self, context):
     def execute(self) -> int:
         errors = 0
 
-        if self.verify_tool_group(self.context.group) != 0:
-            return 1
+        groups = self.context.group.split(",")
+        # groups is never empty and if the user doesn't specify a --group,
+        # then it has the value "default"
+        for group in groups:
+            if self.verify_tool_group(group) != 0:
+                self.logger.warn(f'No such group "{group}".')
+                errors = 1
+                continue
 
-        try:
+            errors, tools_not_found = self._clear_remotes(group)
+
+            if self.context.name and tools_not_found:
+                self.logger.warn(
+                    f"Tools {sorted(tools_not_found)} not found in group {group}"
+                )
+
+            # Remove a custom (non-default) tool group directory if there are
+            # no tools registered anymore under this group
+            if group != "default" and not any(self.tool_group_dir.iterdir()):
```

Comment:
Can you use `is_empty()` here, like you now do at line 70?

(Or, better yet, can you refactor the code there into a subroutine which you can call from here??)